frontend/src/components/lobby/join-lobby/JoinLobby.jsx

```diff
@@ -94,8 +110,17 @@ export default function App() {
                     }
                 />
                 <button className="btn btn-success" onClick={joinLobby}>Csatlakozás</button>
```

Comment:
ezzel akkor is lehet csatlakozni ha nincs lobby
ekkor [object Object] lobbyba csatlakozik ami valószínűleg nem lesz jó